src/handle_append_entries.cxx

```diff
@@ -1062,24 +1069,28 @@ void raft_server::handle_append_entries_resp(resp_msg& resp) {
                           ( cb_func::GotAppendEntryRespFromPeer, &param );
         (void)rc;
 
-        // Try to commit with this response.
-        ulong committed_index = get_expected_committed_log_idx();
-        commit( committed_index );
-
-        // Try enable stream here
+        // Try to enable stream
         int32 max_gap_in_stream = ctx_->get_params()->max_log_gap_in_stream_;
         ulong acceptable_precommit_idx = resp.get_next_idx() +
                                          max_gap_in_stream;
         ulong last_streamed_log_idx = p->get_last_streamed_log_idx();
+        p_tr("max gap: %d, acceptable_precommit_idx: %ld, last_streamed_log_idx: %ld, "
+             "last_sent: %ld, next_idx: %ld", max_gap_in_stream, 
+             acceptable_precommit_idx, last_streamed_log_idx, p->get_last_sent_idx(), 
```

Comment:
We need to check whether it should enable stream first. Because calling commit will change **p->get_last_sent_idx()**